src/Rest/Routes/Settings/BulkRoute.php

```diff
@@ -0,0 +1,388 @@
+	{
+		$error = $details['error'] ?? [];
+		$success = $details['success'] ?? [];
+		$skip = $details['skip'] ?? [];
+
+		$msg = '';
+
+		switch ($type) {
+			case 'sync':
+				$msg = \esc_html__('synced', 'eightshift-forms');
+				break;
+			case 'delete':
+				$msg = \esc_html__('deleted', 'eightshift-forms');
+				break;
+			case 'restore':
+				$msg = \esc_html__('restored', 'eightshift-forms');
+				break;
+			case 'delete-perminentely':
+				$msg = \esc_html__('deleted perminently', 'eightshift-forms');
+				break;
+		}
+
+		if (!$details) {
+			return [
+				'status' => 'error',
+				// translators: %s replaces type.
+				'msg' => \sprintf(\esc_html__('There are no forms in your list to %s.', 'eightshift-forms'), $msg),
+			];
+		}
+
+		if (\count($details) > 1) {
+			$msgOutput = [
+				// translators: %s replaces type.
+				\sprintf(\esc_html__('Not all forms were %s with success. Please check the following log.', 'eightshift-forms'), $msg),
+			];
+
+			if ($error) {
+				// translators: %s replaces error list.
+				$msgOutput[] = \sprintf(\esc_html__('<br/><strong>Error:</strong><br/> %s', 'eightshift-forms'), \implode('<br/>', $error));
+			}
+
+			if ($success) {
+				// translators: %s replaces success list.
+				$msgOutput[] = \sprintf(\esc_html__('<br/><strong>Success:</strong><br/> %s', 'eightshift-forms'), \implode('<br/>', $success));
+			}
+
+			if ($skip) {
+				// translators: %s replaces skip list.
+				$msgOutput[] = \sprintf(\esc_html__('<br/><strong>Skip:</strong><br/> %s', 'eightshift-forms'), \implode('<br/>', $skip));
+			}
+
+			return [
+				'status' => 'warning',
+				'msg' => \implode('<br/>', $msgOutput),
+			];
+		}
+
+		if ($success) {
+			return [
+				'status' => 'success',
+				// translators: %s replaces form msg type.
+				'msg' => \sprintf(\esc_html__('Success, all forms were %s.', 'eightshift-forms'), $msg),
+			];
+		}
+
+		if ($skip) {
+			return [
+				'status' => 'warning',
+				'msg' => \esc_html__('Warning, all forms were skipped.', 'eightshift-forms'),
+			];
+		}
+
+		return [
+			'status' => 'error',
+			'msg' => \esc_html__('There was and error on all forms.', 'eightshift-forms'),
+		];
+	}
+
+	/**
+	 * Sync forms but Ids.
+	 *
+	 * @param array<int> $ids Form Ids.
+	 *
+	 * @return array<int>
+	 */
+	private function sync(array $ids): array
+	{
+		$output = [];
+
+		foreach ($ids as $id) {
+			$title = \get_the_title($id);
+
+			if (!$title) {
+				// translators: %s replaces form id.
+				$title = \sprintf(\esc_html__('Form %s', 'eightshift-forms'), $id);
+			}
+
+			// Prevent non syncahble forms from syncing like mailer.
+			if (!Helper::canIntegrationUseSync(Helper::getFormTypeById((string) $id))) {
+				$output['skip'][] = $title;
+				continue;
+			}
+
+			$item = $this->integrationSyncDiff->syncFormDirect((string) $id);
+
+			$output[$item['status']][] = $title;
+		}
+
+		return $this->output($output, 'sync');
+	}
+
+	/**
+	 * Delete forms but Ids.
```

Comment:
```suggestion
	 * Delete forms by Ids.
```
Please set your doc blocker AI to PEGI 16 😄